### PR TITLE
Added support for Raspbian

### PR DESCRIPTION
**Pull request by mattqm**
#### Pull Request (PR) description
Sets the distro to Debian for Raspbian when subscribing to the influxdb repo.

#### This Pull Request (PR) fixes the following issues

Raspbian is a drop replacement for Debian - it uses an identical package/repo structure.  The current code causes the influxdb download to fail as they do not have Raspbian as an OS, but Debian can be used directly.

### CODE REVIEW

**Comment by mattqm** [comment]
Looks like my second commit caused the checks to fail.  I added Debian 10 to the valid OS's in the metadata.json file as it works fine on my systems.  I'm not sure what causes the tests to fail.

File: metadata.json [hunk 1]
```diff
@@ -19,10 +19,18 @@
         "16.04"
       ]
     },
+    {
+      "operatingsystem": "Raspbian",
+      "operatingsystemrelease": [
```
Comment (dhoppe) [inline metadata.json:26]: Please add support for Raspbian 9 aswell.

**Comment by bastelfreak** [comment]
@mattqm can you please rebase now since https://github.com/voxpupuli/puppet-telegraf/pull/144 got merged?

**Comment by mattqm** [comment]
Ok, resolved the conflict.

**Review by dhoppe — APPROVED** [review]

**Pull request merged**